content/resources/lessons/sound-basics/presentation/index.md

```diff
@@ -91,6 +102,15 @@ $$
 
 {{% /fragment %}}
 
+>Notes:
+>- Decibels are always the log of a ratio.
+>- We typically scale based on a pain threshold of 20 Pa, and a reference value of
+>  0.00002 Pa = 0 dB, which is considered to be the softest sound level a human ear can
+>  detect / near total silence. 
```

Comment:
hmmm, doubt that this is really 0 dB, though it looks like this is not your fault, you're just copying from example above. It looks like `≈0 dB` is more accurate. 

Add a link maybe, this is cool: https://www.epd.gov.hk/epd/noise_education/web/text/ENG_EPD_HTML/m1/intro_5.html#:~:text=Human%20ear%20can%20perceive%20a,called%20the%20Threshold%20of%20Hearing.

Reply:
Mm ok yea, I'm not sure what source I originally used, in your link they also say this:
> the decibel or dB scale is used. The scale uses the hearing threshold of 20 μPa or 20 x 10-6 Pa as the reference level. This is defined as 0 dB.

Can use whichever you think is more appropriate, = or ≈ ?
And I'll add this link, it's nice.